arrays/c-or-cpp/vdec.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>

```

Comment:
Please put the file with the complete name of the algorithm

Reply:
done